src/common/src/util/stream_graph_visitor.rs

```diff
@@ -88,7 +88,12 @@ fn visit_stream_node_tables_inner<F>(
                 always!(node.right_degree_table, "HashJoinDegreeRight");
             }
             NodeBody::DynamicFilter(node) => {
-                always!(node.left_table, "DynamicFilterLeft");
+                if node.condition_always_relax {
+                    always!(node.left_table, "DynamicFilterLeftNotSatisfy");
```

Comment:
Do we test this new branch somehow?

Reply:
The current e2e test includes the branch.